src/cozmo/robot.py

```diff
@@ -1141,6 +1172,28 @@ def go_to_pose(self, pose, relative_to_robot=False):
         self._action_dispatcher._send_single_action(action)
         return action
 
+    def go_to_object(self, object, distance_from_object):
+        '''Tells Cozmo to drive to the specified object.
+
+        Args:
+            object (:class:`cozmo.objects.ObservableObject`): The destination object.
+            distance_from_object (:class:`cozmo.util.Distance`): The distance from
+                the object to stop at - this is the distance between the origins so
+                e.g. from between Cozmo's 2 front wheels to the center of a cube
+                is ~40mm
+        Returns:
+            A :class:`cozmo.robot.GoToObject` action object which can be queried
+                to see when it is complete.
+        '''
+        if not isinstance(object, objects.ObservableObject):
+            raise TypeError("Invalid charger supplied")
```

Comment:
Presumably don't mean charger here, since it can be any  observable object..

Perhaps "Target must be an observable object" instead?